xrootd/client/file.go

```diff
@@ -24,6 +24,14 @@ type file struct {
 	info        *xrdfs.EntryStat
 }
 
+// Name returns the name of the file.
+func (f file) Name() string {
+	if f.info == nil {
+		return ""
+	}
+	return f.info.Name()
```

Comment:
XRootD doesn't return name as part of stat info for the `open` and `stat` requests: http://xrootd.org/doc/dev45/XRdv310.htm#_Toc464248823
We can only guess it from the path provided to the `fileSystem.Open` unfortunately.

Reply:
ah! you know the specs better than I do, now :)
you're officially the expert :P 

ok. I'll drop the xrootd modification and will just cache the path inside `rootio.xrdFile` for now.

thanks!